alws/crud/sign_task.py

```diff
@@ -396,6 +422,14 @@ async def complete_gen_key_task(
     return sign_key
 
 
+def _get_mapping_rpm_per_platform(
+    rpms: list[typing.Union[models.SourceRpm, models.BinaryRpm]],
```

Comment:
please, use type hints from the `typing` module instead, some type hints are not available for older Python versions
We might fall during exports on Pulp host